<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaunchMultiGame</class>
 <widget class="QDialog" name="LaunchMultiGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="rB_Heberger">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>HÃ©berger une partie</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rB_Rejoindre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Rejoindre une partie</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_IP">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Port">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_Fermer">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_OK">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
